### PR TITLE
Hotfix/employees refactor

### PR DESCRIPTION
**Pull request by audipasuatmadi**

### CODE REVIEW

File: src/pages/Employees/Employees.js [hunk 1]
```diff
@@ -25,6 +25,8 @@ export default function Employees() {
   const [editDialogOpened, setEditDialogOpened] = useState(false);
   const [refresh, setRefresh] = useState(1);
 
+  const employeeData = useEmployeesData(refresh);
+
```
Comment (krisnadwipayana07) [inline src/pages/Employees/Employees.js:29]: This is great odin

**Review by krisnadwipayana07 — COMMENTED** [review]
Test

File: src/pages/Employees/Employees.js (hunk 1 above)
Comment (audipasuatmadi) [reply src/pages/Employees/Employees.js:29]: Thanks bomski

**Review by krisnadwipayana07 — APPROVED** [review]
Nicee, look efficient

**Pull request merged**